src/indexer.c

```diff
@@ -234,16 +234,18 @@ static void writeCurEntries(DocumentIndexer *indexer, RSAddDocumentCtx *aCtx, Re
 
 /** Assigns a document ID to a single document. */
 static int makeDocumentId(Document *doc, IndexSpec *spec, int replace, const char **errorString) {
-  const char *keystr = RedisModule_StringPtrLen(doc->docKey, NULL);
+  size_t keyLen;
```

Comment:
Another case where having a function converting a redis string to a doc key would make the code cleaner.

Reply:
replace with  MakeDocKeyR